templates/content/stats.html.twig

```diff
@@ -1,5 +1,24 @@
 {% extends 'layout.html.twig' %}
 
+{% macro barGraph(title, stats) %}
+    <h2 class="content-head is-center">{{ title }}</h2>
+    <div class="chart chart-month">
+        {% if stats %}
+            {% for key, value in stats %}
+                <div class="graph-column">
+                    <div class="graph-counterValue">{{ value }}</div>
+                    <div class="graph-bar" style="height: {{ (value / max(stats)) * 300 }}px;"></div>
+                    <div class="graph-legend-x">{{ key|capitalize }}</div>
+                </div>
+            {% endfor %}
+        {% else  %}
```

Comment:
You could use for / else feature

Reply:
Indeed, I wanted to do that before but forgot and just copy/paste this block :smile:

Reply:
(done)